modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -241,6 +253,9 @@ public class TableManager extends Producer<TableEvent, TableEventParameters> imp
     /** Resolver that resolves a network address to cluster node. */
     private final Function<NetworkAddress, ClusterNode> clusterNodeResolver;
 
+    private final Map<UUID, Map<UUID, IndexStorageAdapterFactory>> indexStorageAdapterFactories = new ConcurrentHashMap<>();
+    private final Map<UUID, Map<UUID, IndexLockerFactory>> indexLockerFactories = new ConcurrentHashMap<>();
```

Comment:
Same as above it's either a part of index of table, depending of whether we consider indexes as a lock-aware structures.

Reply:
moved to a table